### PR TITLE
Copy over License/Owners

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - codificat
+      - iranzo
+      - ptrnull
+    emeritus_approvers:
+      - karmab # 10 july 2019
+      - tripledes # 10 july 2019
+      - e-minguez # 10 july 2019
+      - joeldavis84 # 10 july 2019
+
+  "^_posts/.*":
```
Comment (codificat) [inline OWNERS]: We probably don't need that in this repo, right?

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    labels:
+      - kind/blog
```
Comment (codificat) [inline OWNERS]: ditto

File: OWNERS [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "^_(data|layouts|sass|css|js)/.*":
+    labels:
+      - kind/website
```
Comment (codificat) [inline OWNERS]: ditto

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 jekyll
```
Comment (codificat) [inline LICENSE]: This comes from the website.

Shouldn't we use a more content-oriented license here? CC-by-sa?

**Review by codificat — CHANGES_REQUESTED** [review]
These are a copy&paste from the website but I believe some changes are in order - thoughts welcome.

File: OWNERS (hunk 2 above)
Comment (iranzo) [reply OWNERS]: done

File: LICENSE (hunk 4 above)
Comment (iranzo) [reply LICENSE]: done

File: OWNERS (hunk 3 above)
Comment (iranzo) [reply OWNERS]: done

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+     the licensed material may still be restricted for other
+     reasons, including because others have copyright or other
+     rights in the material. A licensor may make special requests,
+     such as asking that all changes be marked or described.
+     Although not required by our licenses, you are encouraged to
+     respect those requests where reasonable. More considerations
+     for the public:
+    wiki.creativecommons.org/Considerations_for_licensees
+
+=======================================================================
```
Comment (codificat) [inline LICENSE:55]: IANAL but I believe this block is not part of the license itself, and therefore it should be removed.

Comment (iranzo) [reply LICENSE:55]: IANAL either, got them from : https://creativecommons.org/2014/01/07/plaintext-versions-of-creative-commons-4-0-licenses/

**Comment by codificat** [comment]
If you look at https://creativecommons.org/licenses/by-sa/4.0/legalcode you'll
see what I mean: the text of the license itself is what's delimited by
`===` in the txt version, right?

File: LICENSE (hunk 5 above)
Comment (codificat) [reply LICENSE:55]: If you look at https://creativecommons.org/licenses/by-sa/4.0/legalcode you'll see what I mean: the text of the license itself is what's delimited by `===` in the txt version, right?

Comment (codificat) [reply LICENSE:55]: Looking closer, we might prefer CC-BY... e.g. the Kubernetes website uses it and its [LICENSE](https://github.com/kubernetes/website/blob/master/LICENSE) file is a full copy, including that section.

Should we use that?

Comment (iranzo) [reply LICENSE:55]: Pulled the raw file from them

**Pull request merged**